install/kubernetes/helm/istio/charts/tracing/templates/service.yaml

```diff
@@ -31,12 +31,12 @@ items:
       release: {{ .Release.Name }}
       heritage: {{ .Release.Service }}
   spec:
+    type: {{ .Values.service.type }}
```

Comment:
As per change below, this should become `.Values.service.uiType` - as distinct from the service type of the zipkin service.

Reply:
Ah, I see, then I'd like to keep here as unchanged, just hardcode the service type as loadbalancer for now.